packages/aws-cdk/lib/api/util/cloudformation.ts

```diff
@@ -400,7 +400,6 @@ async function uploadBodyParameterAndCreateChangeSet(
       env.resolvedEnvironment,
       new AssetManifestBuilder(),
       env.resources,
-      env.sdk,
```

Comment:
It doesn't need the SDK anymore because:

https://github.com/aws/aws-cdk/blob/740db432814bd8448d5898170650bd8961277f47/packages/aws-cdk/lib/api/util/template-body-parameter.ts#L126-L132